packages/client/hmi-client/src/services/charts.ts

```diff
@@ -32,6 +32,185 @@ export interface HistogramChartOptions extends BaseChartOptions {
 	variables: { field: string; label?: string; width: number; color: string }[];
 }
 
+export interface ErrorChartOptions extends Omit<BaseChartOptions, 'height' | 'yAxisTitle' | 'legend'> {
+	maxBins?: number;
+	height?: number;
+	areaChartHeight?: number;
+	boxPlotHeight?: number;
+	variables: { field: string; label?: string }[];
+}
+
+export const createErrorChart = (dataset: Record<string, any>[], options: ErrorChartOptions) => {
+	const axisColor = '#EEE';
+	const labelColor = '#667085';
+	const labelFontWeight = 'normal';
+	const globalFont = 'Figtree';
+
+	const areaChartColor = '#1B8073';
+	const dotColor = '#67B5AC';
+	const boxPlotColor = '#000';
+
+	const width = options.width;
+	const height = options.height ?? 100;
+	const boxPlotHeight = options.boxPlotHeight ?? 16;
+	const areaChartHeight = options.areaChartHeight ?? 44;
+	const gap = 15;
+
+	const areaChartRange = [areaChartHeight, 0];
+	const dotChartRange = [areaChartHeight + gap, areaChartHeight + gap + boxPlotHeight];
+	const boxPlotYPosition = (dotChartRange[0] + dotChartRange[1]) / 2;
+
+	const variablesOptions = options.variables.map(({ field, label }) => ({ field, label: label ?? field }));
+
+	const variables = variablesOptions.map((v) => v.field);
+	const maxbins = options.maxBins ?? 30;
+
+	const titleObj = options.title
+		? {
+				text: options.title,
+				anchor: 'start',
+				subtitle: ' ',
+				subtitlePadding: 4
```

Comment:
Why adding an empty space subtitle and a padding if there are none?

Reply:
Probably it's to add padding below the main title.